client/src/components/Text/Text.stories.tsx

```diff
@@ -0,0 +1,206 @@
+      type: 'string',
+    },
+    size: {
+      type: 'string',
+      options: sizes,
+      control: {
+        type: 'select',
+      },
+      table: {
+        type: {
+          summary: 'TextSize',
+        },
+        defaultValue: {
+          summary: 'md',
+        },
+      },
+    },
+    v: {
+      type: 'string',
+      options: variants,
+      control: {
+        type: 'select',
+      },
+      table: {
+        type: {
+          summary: 'TextVariant',
+        },
+        defaultValue: {
+          summary: 'body1',
+        },
+      },
+    },
+    color: {
+      type: 'string',
+      options: colors,
+      control: {
+        type: 'select',
+      },
+      table: {
+        type: {
+          summary: 'TextColor',
+        },
+        defaultValue: {
+          summary: 'primary',
+        },
+      },
+    },
+    align: {
+      type: 'string',
+      options: alignments,
+      control: {
+        type: 'select',
+      },
+      table: {
+        type: {
+          summary: 'TextAlign',
+        },
+        defaultValue: {
+          summary: 'left',
+        },
+      },
+    },
+    multiline: {
+      type: 'boolean',
+      control: {
+        type: 'boolean',
+      },
+    },
+    ellipsis: {
+      type: 'boolean',
+      control: {
+        type: 'boolean',
+      },
+    },
+    loading: {
+      type: 'boolean',
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
+}
+
+export default meta
+
+type Story = StoryObj<typeof Text>
+
+const initVariants = <P extends keyof TextProps>(prop: P, items: TextProps[P][]) => (
+  <Block style={{ display: 'flex', flexDirection: 'column' }}>
+    {items.map((item) => <Text content={item} {...{ [prop]: item }}/>)}
+  </Block>
+)
+
+const showDescription = (value: string) => {
+  const description = {
+    docs: {
+      description: {
+        story: value,
+      },
+    },
+  }
+
+  return description
+}
+
+export const Default: Story = {
+  args: {
+    content: 'Text',
+    v: 'body1',
+    size: 'md',
+  },
+  parameters: showDescription('The default variant of the Text component.'),
+}
+
+export const Variants: Story = {
+  parameters: showDescription('The variants of the Text component.'),
+  decorators: [
+    () => initVariants('v', variants),
+  ],
+}
+
+// TODO: [kseniya_boldak] fix size application
+export const Sizes: Story = {
+  parameters: showDescription('The size of the Text component.'),
+  decorators: [
+    () => initVariants('size', sizes),
+  ],
+}
+
+export const Colors: Story = {
+  parameters: showDescription('The color of the Text component.'),
+  decorators: [
+    () => initVariants('color', colors),
+  ],
+}
+
+export const Alignments: Story = {
+  parameters: showDescription('The alignment of the text in an element.'),
+  decorators: [
+    () => initVariants('align', alignments),
+  ],
+}
+
+export const Multiline: Story = {
+  parameters: showDescription('If true the text is shown in multiple lines.'),
```

Comment:
By default we show one line text. If we need multiline we can set multiline flag